sbnanaobj/StandardRecord/SRTrackCalo.h

```diff
@@ -21,10 +21,15 @@ namespace caf
       float dedx; //!< dQ/dx [ADC/cm] -- pre calibration and electron lifetime correction
       float pitch; //!< Track pitch [cm]
       float t; //!< Time of deposition [ticks]
-      SRVector3D p; //!< Position of deposition [cm]      
+      // NOTE: flatten-caf seems to have trouble with nesting the point
```

Comment:
Do we need to worry about this nesting elsewhere?

Reply:
Not sure... I think it's because the SRCaloPoint is inside an SRTrackCalo which is inside an array, so it's kind of a corner case.

Reply:
We should probably be on the lookout this level of nesting elsewhere (maybe that would make sense as a GitHub "Issue" @wesketchum?), but in any case this shouldn't hold up this particular PR.